### PR TITLE
chore: Added horizontal-logo-light and Storybook, #issue 27

### PR DESCRIPTION
**Pull request by niyashiyas**
### Check List

- [x] I have read the [Contributing Guidelines](https://github.com/nodejs/nodejs.org/blob/main/CONTRIBUTING.md) and made commit messages that follow the guideline.
- [x] I have run `npx turbo lint` to ensure the code follows the style guide. And run `npx turbo lint:fix` to fix the style errors if necessary.
- [x] I have run `npx turbo format` to ensure the code follows the style guide.
- [x] I have run `npx turbo test` to check if all tests are passing.
- [x] I've covered new added functionality with unit tests if necessary.

### CODE REVIEW

File: components/__design__/horizontal-logos.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,29 @@
+import Image from 'next/image';
+import type { Meta as MetaObj, StoryObj } from '@storybook/react';
+
+export const HorizontalLogos: StoryObj = {};
+
+export default {
+  title: 'Design System/Logos',
+  component: () => {
+    return (
+    <div>
```
Comment (HinataKah0) [inline components/__design__/horizontal-logos.stories.tsx]: Nit: please add a tab so it looks like this:
```js
component: () => {
  return (
    <div>
    ...
    </div>
  );
},
```

And don't forget to fix the indentation in `<Image />` below.

File: components/__design__/horizontal-logos.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+        <Image
+            src={`/static/images/logos/horizontal-dark.svg`}
+            alt="Node.js"
+            width={267}
+            height={80}
+          />
+      </div>
+    </div>
+  )},
+} as MetaObj;
```
Comment (HinataKah0) [inline components/__design__/horizontal-logos.stories.tsx]: Nit: don't forget to add a newline

**Review by HinataKah0 — COMMENTED** [review]
Left 2 minor comments.
Thank you for the contribution! 🎉

File: components/__design__/horizontal-logos.stories.tsx [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+        <Image
+            src={`/static/images/logos/horizontal-light.svg`}
+            alt="Node.js"
+            width={267}
+            height={80}
+          />
+      </div>
+      <div>
+        <Image
+            src={`/static/images/logos/horizontal-dark.svg`}
```
Comment (HinataKah0) [inline components/__design__/horizontal-logos.stories.tsx]: Let's not displaying `horizontal-dark.svg` first because the file hasn't been added.

**Comment by AugustinMauroy** [comment]
There are an stranger things
Capture d’écran 2023-09-24 à 10 56 03

Here an fix svg:

Frame
Frame-2

**Comment by niyashiyas** [comment]
All the required changes were made in Storybook 
Horizontal-dark logo was added ,
Horizontal-light logo is updated. 
Can you please review it again

File: components/__design__/horizontal-logos.stories.tsx [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+import type { Meta as MetaObj, StoryObj } from '@storybook/react';
+
+export const HorizontalLogos: StoryObj = {};
+
+export default {
+  title: 'Design System/Logos',
+  component: () => {
+    return (
+      <div>
+        <div>
```
Comment (AugustinMauroy) [inline components/__design__/horizontal-logos.stories.tsx]: You can remove useless `div`.

File: components/__design__/horizontal-logos.stories.tsx [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+          <Image
+              src={`/static/images/logos/horizontal-light.svg`}
```
Comment (AugustinMauroy) [inline components/__design__/horizontal-logos.stories.tsx]: Adding black background allow us to seed the logo 😁
```suggestion
              src={`/static/images/logos/horizontal-light.svg`}
               className="bg-black"
```

File: components/__design__/horizontal-logos.stories.tsx (hunk 2 above)
Comment (AugustinMauroy) [reply components/__design__/horizontal-logos.stories.tsx]: FYI: you can use
```bash
npx turbo format
```

**Review by AugustinMauroy — COMMENTED** [review]
Good job. Small improvement need but your are going forward.

**Review by AugustinMauroy — APPROVED** [review]
LGTM

**Review by aymen94 — APPROVED** [review]

File: components/__design__/horizontal-logos.stories.tsx [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+export const HorizontalLogos: StoryObj = {};
+
+export default {
+  title: 'Design System/Logos',
+  component: () => {
+    return (
+      <div>
+        <Image
+          src={`/static/images/logos/horizontal-light.svg`}
+          className="bg-black"
```
Comment (bmuenzenmeyer) [inline components/__design__/horizontal-logos.stories.tsx:13]: I'm not sure we want this here - it illustrates the contrast but doesn't show the image in its raw form - which is a departure from the rest of the work I've seen so far. Opinions @nodejs/nodejs-website ?

```suggestion
```

Comment (bmuenzenmeyer) [reply components/__design__/horizontal-logos.stories.tsx:13]: image

https://www.chromatic.com/review?appId=64c7d71358830e9105808652&number=5879&type=linked&view=changes

Comment (AugustinMauroy) [reply components/__design__/horizontal-logos.stories.tsx:13]: For me add a dark background is cool

**Comment by bmuenzenmeyer** [comment]
hearing no other dissent, we can always address later

**Pull request merged**